extension/gulpfile.js

```diff
@@ -87,17 +97,29 @@ gulp.task('browserify', () => {
     'app/src/report-loader.js'
   ], {read: false})
     .pipe(tap(file => {
-      file.contents = browserify(file.path, {
-        transform: ['brfs']
+      let bundle = browserify(file.path, {
+        transform: ['brfs'],
       })
       // Do the additional transform to convert references to devtools-timeline-model
       // to the modified version internal to Lighthouse.
       .transform('./dtm-transform.js', {
         global: true
       })
       .ignore('npmlog')
-      .ignore('chrome-remote-interface')
-      .bundle();
+      .ignore('chrome-remote-interface');
+
+      const srcPath = /\.\.\/src\//;
+
+      // Expose the audits and gatherers so they can be dynamically loaded.
+      bundle = audits.reduce((bundle, audit) => {
```

Comment:
unless this is a hot path, the readability and expressiveness gains of  reduce are always a :+1: to me